mods/cnc/rules/ships.yaml

```diff
@@ -70,6 +70,8 @@ LST:
 	RevealsShroud:
 		Range: 7c0
 	WithFacingSpriteBody:
+	Selectable:
+		Bounds: 48,40
```

Comment:
IMO this should be 48,48: this matches the original sprite, and looks better across the range of casings.